### PR TITLE
Created Basic Web Structure

### PR DESCRIPTION
**Pull request by gitumarkk**
- Created header and footer for page but further styling needed
- Added pages : Home, About Us, Registration Closed, Resources, Course Schedule and Contact Us)
- Added CSS style sheets (960.css (grid stylesheet as the bootstrap grid system is not the best), bootstrap.min, reset.css and umonya (custom css))
- Added Javascript (jquery.min, bootstrap.min, bootstrap-carousel.js (for the slider), umonya.js (custom umonya js).
- Created slider for home page (temporary though)
- Deleted extra sets of images

### CODE REVIEW

File: umonya/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Django == 1.5.1
```
Comment (Rizziepit) [inline umonya/requirements.txt]: no spaces e.g. `Django==1.5.1`

**Comment by Rizziepit** [comment]
@gitumarkk can you change this PR to merge into the develop branch?

**Comment by Rizziepit** [comment]
rename `app` folder in umonya/umonya/ to `apps` - we will be putting multiple apps in there

File: umonya/umonya/settings.py [hunk 2]
```diff
@@ -1,4 +1,11 @@
 # Django settings for umonya project.
+import os
+
+#~ Directories Path for Project
+
+UMONYA_BASE_DIR 		= os.path.abspath(os.path.dirname(__file__))
+UMONYA_TEMPLATE_DIR		= os.path.abspath(os.path.join(os.path.dirname( __file__ ),'templates'))
+UMONYA_STATIC_DIR		= os.path.abspath(os.path.join(os.path.dirname( __file__ ), 'static'))
```
Comment (Rizziepit) [inline umonya/umonya/settings.py]: you should instead write this as `UMONYA_TEMPLATE_DIR = os.path.join(UMONYA_BASE_DIR, 'templates')`

File: umonya/umonya/settings.py [hunk 3]
```diff
@@ -11,8 +18,8 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
-        'NAME': '',                      # Or path to database file if using sqlite3.
+        'ENGINE': 'django.db.backends.sqlite3', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
```
Comment (Rizziepit) [inline umonya/umonya/settings.py:21]: @gitumarkk @ninas @LyndsayLawrence @BradlyLawrence sqlite3 is good to start with, and for testing, but we'll have to move to something else. MySQL or Postgres?

File: umonya/umonya/settings.py [hunk 4]
```diff
@@ -120,6 +129,7 @@
     'django.contrib.sites',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'umonya.app.main',
```
Comment (Rizziepit) [inline umonya/umonya/settings.py]: @gitumarkk move this app to the top of `INSTALLED_APPS`. That way the main app can selectively override the Django templates with its own.

File: umonya/umonya/urls.py [hunk 5]
```diff
@@ -6,7 +6,13 @@
 
 urlpatterns = patterns('',
     # Examples:
-    # url(r'^$', 'umonya.views.home', name='home'),
+    url(r'^$', 'umonya.app.main.views.home', name='home'),
```
Comment (Rizziepit) [inline umonya/umonya/urls.py]: @gitumarkk rather declare these urls in `main/urls.py`. then use `include(umonya.app.main.urls)` here

File: umonya/umonya/settings.py (hunk 3 above)
Comment (gitumarkk) [inline umonya/umonya/settings.py:21]: @Rizziepit I am more familiar with MySQL (never used Postgres), so probably best for me to use it.

**Comment by gitumarkk** [comment]
@Rizziepit I am not sure how to change the PR to merge into the develop branch, i.e. I followed the steps given in "https://help.github.com/articles/merging-a-pull-request" but I am unable to see the merge button.

**Comment by Rizziepit** [comment]
@gitumarkk seems github won't allow you to swap the base branch for a PR. so you have to open a new pull request. I see you've opened a new one, but it is still requesting a merge into the master branch. close it and open a new one? you might have to do a pull in your repo to get the develop branch I created

**Pull request closed without merging**